samples/crypto/psa_tls/sample.yaml

```diff
@@ -138,6 +175,35 @@ tests:
       - nrf54l15dk/nrf54l15/cpuapp/ns
     tags: ci_build cracen sysbuild ci_samples_crypto
   ################################################################################
+  ## PSA APIs with Cracen and Oberon
+  ################################################################################
+  sample.psa_tls.1_3_server.ecdsa.cracen_oberon:
+    sysbuild: true
+    build_only: true
+    extra_args: >
+      OVERLAY_CONFIG="overlays/server.conf;overlays/ecdsa.conf;overlays/cracen-oberon-psa.conf;overlays/tls_13.conf"
+    platform_allow: >
+      nrf54l15dk/nrf54l15/cpuapp nrf54l15pdk/nrf54l15/cpuapp
+      nrf54l15dk/nrf54l15/cpuapp/ns
+    integration_platforms:
+      - nrf54l15dk/nrf54l15/cpuapp
+      - nrf54l15pdk/nrf54l15/cpuapp
```

Comment:
PDK, really? :thinking:

Reply:
Pdk support will be properly removed in a future PR from what I understood

Reply:
PDK support has been removed now